src/universal/conf/ethereumHF.conf

```diff
@@ -18,7 +18,7 @@ mantis {
     eip150-block-number = "2463000"
     eip155-block-number = "2675000"
     eip160-block-number = "2675000"
-    # TODO: EIP-161 a,b,c,d = 2675000
+    eip161-block-number = "2675000"
     # TODO: EIP-170
```

Comment:
Oh, could you remove the `TODO` apparently we don't need a block number for that (should have been done earlier) ?

Reply:
hmm Good catch. I think there should be `max-code-size` specified, in EIP170 pr. Am I right @ntallar ?

Reply:
Added here `max-code-size =  24576` and removed todo